app/views/layouts/_menu.html.haml

```diff
@@ -1,40 +1,41 @@
-#sidebar
-  #profile.user-sidebar
-    %img.profileimg{:src => "http://0.gravatar.com/avatar/59b762ebf3d116bf806bcfa3fa12e65f"}/
-    %p.name Hello, woubuc
-    .row.links.hidden-sm.hidden-xs
-      .col-xs-6
-        %a{:href => ""} Account
-      .col-xs-6
-        %a.right{:href => "#", :onclick => 'EpicLogger.logout()'} Log out
-    %button.btn.btn-primary.fa.fa-bars.visible-sm.visible-xs.pull-right{"data-toggle" => "collapse", :href => "#collapse_menu"}
-  #collapse_menu.collapse.in
-    .selectbox.closed
-      %p.picked-website
-        %strong.title Current Site Name
-        %span.closed.fa.fa-chevron-down
-        %span.open.fa.fa-chevron-up
-      %ul#websites-sidebar.options
-        = nav_link "Some website", '#', {:class=>'title picked', 'data-id'=>'', 'onclick'=>'EpicLogger.pickWebsite(this)'}
-      #add-new.nav.nav-pills.nav-stacked
-        = link_to new_website_path(), class: 'title' do
-          = icon('globe')
-          Add new website
-    %ul.nav.nav-pills.nav-stacked
-      = nav_link "Errors", errors_url(), {}, 'exclamation-triangle'
-    %ul.nav.nav-pills.nav-stacked
-      = nav_link "Installation", installations_url(), {}, 'code'
-      = nav_link "Websites", websites_url(), {}, 'briefcase'
-      / = nav_link "Website Settings", settings_url(), {}, 'cog'
-      = nav_link "Members", website_members_url(), {}, 'users'
-    %ul.nav.nav-pills.nav-stacked.hidden-md.hidden-lg
-      %li#menu_account
-        %a{:href => "#"}
-          %span.fa.fa-user
-          Account
-      %li#menu_signout
-        %a{:href => "#"}
-          %span.fa.fa-sign-out
-          Log out
-    %ul.nav.nav-pills.nav-stacked
-      = nav_link "Help", 'mailto:info@epiclogger.com', {'data-helpful'=>'epiclogger','data-helpful-modal'=>"on", :class=>'helpful-link'}, 'question'
\ No newline at end of file
+- unless params[:controller] + '/' + params[:action] == 'websites/new'
```

Comment:
decat sa faci concatenare ca sa verifici asta aici poti sa verifici fiecare parametru in parte. Nu inteleg logica unei concatenari care consuma mai multa memorie cand ai putea sa faci 0 concatenari :))